app/Controller/PagesController.php

```diff
@@ -74,3 +120,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
+        if ($a == $b)  {
```

Comment:
__[staging][code_sniffer]__
* Expected "if (...) {\n"; found "if (...)  {\n"
* Operator == prohibited; use === instead
* There must be a single space between the closing parenthesis and the opening brace of a multi-line IF statement; found 2 spaces